physics/smoke_dust/rrfs_smoke_wrapper.meta

```diff
@@ -916,6 +916,14 @@
   type = real
   kind = kind_phys
   intent = in
+[totprcp]
```

Comment:
Same question. Please remove this change if you're not using `totprcp`.

Reply:
@grantfirl I will remove this unused variable. Thanks.